Clients/GUI/Pages/Confirm.razor

```diff
@@ -0,0 +1,35 @@
+﻿@page "/confirm/{secretToken}"
+
+<div class="col-12">
+    <h3>Confirmation</h3>
+    <hr />
+    @if (result)
+    {
+        <p>Your email was confirmed!</p>
```

Comment:
Maybe add link to main page?

Reply:
We have a link to the main page in the header